Scripts/cert.config

```diff
@@ -0,0 +1,19 @@
+[ ca ]
+default_ca    = CA_default
+[ CA_default ]
+default_md    = sha256
+[ v3_ca ]
+subjectKeyIdentifier=hash
+authorityKeyIdentifier=keyid:always,issuer
+basicConstraints = critical,CA:true
+keyUsage=critical,keyCertSign
+extendedKeyUsage = serverAuth,clientAuth
+[ req ]
+prompt = no
+distinguished_name    = req_distinguished_name
+[ req_distinguished_name ]
+C=RU
+L=RU
+O=Catbird
+CN=Catbird
```

Comment:
here you probably need to specify the host?

Reply:
Yes, but we do not have website.

Reply:
Changed to `http://localhost`